src/fairchem/core/models/equiformer_v2/equiformer_v2.py

```diff
@@ -683,6 +684,11 @@ def no_weight_decay(self) -> set:
 
 @registry.register_model("equiformer_v2_backbone")
 class EquiformerV2Backbone(EquiformerV2, BackboneInterface):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.energy_block = None
```

Comment:
can you add a comment to get rid of this when we stop inheriting from EquiformersV2?